homeassistant/components/pterodactyl/entity.py

```diff
@@ -33,7 +33,7 @@ def __init__(
             name=self.game_server_data.name,
             model=self.game_server_data.name,
             model_id=self.game_server_data.uuid,
-            configuration_url=f"{config_entry.data[CONF_URL]}/server/{identifier}",
+            configuration_url=f"{config_entry.data[CONF_URL].strip('/')}/server/{identifier}",
```

Comment:
Instead of doing magic with fixing URLs by handling them as strings, we should use YARL to build the URL instead.

../Frenck